service/confirmation-repeater/src/index.ts

```diff
@@ -4,17 +4,22 @@ const amqplib = require('amqplib');
 const { Level } = require("level");
 const schedule = require('node-schedule');
 //const nodeSchedule = require("@types/node-schedule")
-const db = new Level(process.env.DB_PATH || "./db", { valueEncoding: 'json' })
-
-const dotenv = require('dotenv');
-dotenv.config();
+const db = new Level(process.env.DB_PATH || "./repeater.db", { valueEncoding: 'json' })
```

Comment:
@1v4n4 a bit more explicit default name then "db" is better. what if it's created in a dir with many other files - it would get a bit lost.